charts/osm/templates/osm-deployment.yaml

```diff
@@ -51,6 +51,8 @@ spec:
               containerPort: 15128
             - name: "metrics"
               containerPort: 9091
+            - name: "osm-config-validator"
```

Comment:
nit: should we name it "osm-webhook-validator" instead?

Reply:
I just re-used the name that's being referenced here: https://github.com/openservicemesh/osm/blob/main/charts/osm/templates/osm-service.yaml#L23